gen-typescript-local-dist.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+# Use this script to generate local TS package files for use in local developement
+
+# Final Step: 
+# navigate to the the local project directory where you want to use 
+# this snapshot and run: yarn link @guardian/bridget
+# More info: https://classic.yarnpkg.com/lang/en/docs/cli/link/
+
+# generate TypeScript Thrift files
+node_modules/.bin/thrift-typescript --target thrift-server --rootDir . --sourceDir thrift --outDir dist/bridget --strictUnions ../thrift/native.thrift
+
+cd dist 
+cd bridget
```

Comment:
not important really, but maybe we could do this in 1 step!

```suggestion
cd dist/bridget
```